modules/python/src2/cv2.cpp

```diff
@@ -681,26 +699,25 @@ PyObject* pyopencv_from(const UMat& m) {
     return o;
 }
 
-template<>
-bool pyopencv_to(PyObject *o, Scalar& s, const char *name)
+static bool pyopencv_to(PyObject *o, Scalar& s, const ArgInfo info)
 {
     if(!o || o == Py_None)
         return true;
     if (PySequence_Check(o)) {
-        PyObject *fi = PySequence_Fast(o, name);
+        PyObject *fi = PySequence_Fast(o, (const char *)info);
         if (fi == NULL)
             return false;
         if (4 < PySequence_Fast_GET_SIZE(fi))
         {
-            failmsg("Scalar value for argument '%s' is longer than 4", name);
+            failmsg("Scalar value for argument '%s' is longer than 4", (const char *)info);
```

Comment:
`info.name` is more clear and safe.

"Hack" operator should not be used at all (it is here for some build problems only) - it candidate for removal.